ui/raidboss/data/06-ew/raid/p9n.txt

```diff
@@ -1,6 +1,259 @@
 ### P9N: Anabaseios: The Ninth Circle
 
+# Ability IDs that appear in this instance:
+# UMFB = Unaspected, Mage, Fighter, Behemoth
+# "Unaspected" being the state the boss is in before the first Ravening, and between Disgorge and Ravening
+#
+# Ability ID   | UMFB | Description
+# Kokytos:8115 | U    | Auto-attack (unaspected)
+# Kokytos:8116 | U F  | Gluttony's Augur - cast bar
+# Kokytos:8117 | U F  | Gluttony's Augur - damage
+# Kokytos:8118 | U  B | Ravening - move to mage phase
+# Kokytos:8119 |  M   | Ravening - move to fighter phase
+# Kokytos:811A |   F  | Ravening - move to Behemoth phase
+# Kokytos:811C |  M   | Auto-attack (magical, mage)
```

Comment:
Can you ignore all of the boss autos from the timeline? (They're generally noisy but also have been inconsistent in other instances.)  If you haven't seen it, you can use `-ii 811C` to ignore an id on the command line when using `make_timeline`.  When writing a timeline, I try to put the command line I used at the top (e.g. golbez-ex.txt has a good example) so other folks rerunning `make_timeline` later for whatever reason can not have to manually redo that work.

Reply:
Can do, I saw some conflicting things in different places on whether to include autos or not, so I decided to include them for now and remove them if they weren't needed.

Should I still include the first auto to have something to sync with near the start? Otherwise the first ability I found near the start is Gluttony's Augur around 11-15 seconds in.

Reply:
Yes, inclusion of the first auto for a safety sync is standard. There have been other Very Specific situations where we include one or two for Reasons, but in general we put them on the ignore list and then specifically include the first one.

Reply:
> Can do, I saw some conflicting things in different places on whether to include autos or not, so I decided to include them for now and remove them if they weren't needed.

For me I usually think:
* keep the first auto (if it's consistent?) to start the timeline
* keep tank autos if they're mechanics (DSR, TOP, maybe Diamond Extreme should)
* `-ii` other autos because they're noisy / not always consistent in older content

The other thing you can do is specifically include the cast time of the first boss ability as well with a larger sync to get things back on track.  The `golbez-ex.txt` file also does this at 9.4 if you want an example.

@JLGarber one thing that had come up previously is maybe just auto-starting timelines (where it makes sense, i.e. raids but not dungeons) when combat starts rather than on engage or auto which would make things more consistent.  Now that we have in combat lines in OverlayPlugin, we could use those instead? Not a thing to do here, but a thing to consider for the future.

Reply:
Added `-ii`s and ignored all the autos and the cast resolutions that didn't seem useful for syncing.